packages/umi-ui-tasks/src/ui/components/Analyze/Treemap/index.tsx

```diff
@@ -0,0 +1,91 @@
+import { Component } from 'react';
+import FoamTree from './carrotsearch.foamtree';
```

Comment:
```suggestion
import FoamTree from './carrotsearch-foamtree';
```

易于辩识，不然以为是 `. foamtree` 文件